server/start.go

```diff
@@ -631,7 +596,15 @@ func startApp(svrCtx *Context, appCreator types.AppCreator, opts StartCmdOptions
 		return app, traceCleanupFn, err
 	}
 
-	app = appCreator(svrCtx.Logger, db, traceWriter, svrCtx.Viper)
+	if isTestnet, ok := svrCtx.Viper.Get(KeyIsTestnet).(bool); ok && isTestnet {
```

Comment:
So what is the relationship between the `start` command, which checks `KeyIsTestnet` and the new testnet command?

Reply:
im not sure this needs to exist. I may be missing something but this tool should only modify the underlying db.